src/commands/common/get-latest-service-agreement/blockchain-get-latest-service-agreement.js

```diff
@@ -91,13 +95,6 @@ class BlockchainGetLatestServiceAgreement extends Command {
             };
         }
 
-        if (latestBlockchainTokenId < latestDbTokenId) {
```

Comment:
Why did we remove this?

Reply:
Duplicate condition there is the same one right before that